root/area/edit_form.tt

```diff
@@ -42,6 +42,16 @@
 
 <script type="text/javascript">//<![CDATA[
   (function () {
+    const wasEnded = $('#id-edit-area\\\.period\\\.ended').prop('checked');
```

Comment:
It would definitely be nice to move this to a module, e.g. root/static/scripts/edit/utility/toggleEnded.js or similar. Then require it from root/static/scripts/edit.js and it'll be loaded on all the edit pages.

The only variation seems to be the entity type, and you can detect it from `document.location.pathname` if nothing else.

If you wrap everything in a [jQuery "ready" callback](https://learn.jquery.com/using-jquery-core/document-ready/), the inputs should exist when it in runs, even if the script is included above those elements.

Placing the code in a file has the added benefit that we can safely use `const`, like you did. As an inline script, here it will break in some of our supported browsers. But the bundled files are [transformed using Babel](https://babeljs.io/repl/#?babili=false&evaluate=true&lineWrap=false&presets=latest%2Creact%2Cstage-2&experimental=false&loose=false&spec=false&code=const%20foo%20%3D%201%3B&playground=true), when `compile_resources.sh` is run, so it'll get converted into something compatible in older browsers (a var statement) while still giving us the compile-time benefits (an error will be thrown if Babel sees you tried to re-assign `wasEnded`).